google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITClosedSessionTest.java

```diff
@@ -221,18 +223,21 @@ public Void run(TransactionContext transaction) throws Exception {
   public void testReadWriteTransactionNoRecreation() {
     client.setAllowSessionReplacing(false);
     client.invalidateNextSession();
-    expectedException.expect(SessionNotFoundException.class);
-    TransactionRunner txn = client.readWriteTransaction();
-    txn.run(
-        new TransactionCallable<Void>() {
-          @Override
-          public Void run(TransactionContext transaction) throws Exception {
-            try (ResultSet rs = transaction.executeQuery(Statement.of("SELECT 1"))) {
-              rs.next();
+    try {
+      TransactionRunner txn = client.readWriteTransaction();
+      txn.run(
+          new TransactionCallable<Void>() {
+            @Override
+            public Void run(TransactionContext transaction) throws Exception {
+              try (ResultSet rs = transaction.executeQuery(Statement.of("SELECT 1"))) {
+                rs.next();
+              }
+              return null;
             }
-            return null;
-          }
-        });
+          });
+    } catch (SessionNotFoundException ex) {
```

Comment:
Add  a `fail("Expected exception")` to prevent false positives.

Reply:
Done